Cannot install Zowe from the source

Hello, I have problems installing Zowe from the source based on the documentation.

If I try to install it, I run into a following problem with a missing package version:

```npm
npm ERR! Can't install C:\Users\ab634601\Documents\development\sandbox\zowe\zowe-cli: Missing package version

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ab634601\AppData\Roaming\npm-cache\_logs\2020-11-09T12_05_47_569Z-debug.log

```

If I add the package version to `package.json`, I then get a following issue:

```npm
npm ERR! code ENOENT
npm ERR! syscall rename
npm ERR! path C:\Users\ab634601\AppData\Roaming\npm\node_modules\.staging\@zowe\cli-1b8edd5f\node_modules\@babel\code-frame
npm ERR! dest C:\Users\ab634601\AppData\Roaming\npm\node_modules\.staging\@babel\code-frame-4adefc22
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, rename 'C:\Users\ab634601\AppData\Roaming\npm\node_modules\.staging\@zowe\cli-1b8edd5f\node_modules\@babel\code-frame' -> 'C:\Users\ab634601\AppData\Roaming\npm\node_modules\.staging\@babel\code-frame-4adefc22'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ab634601\AppData\Roaming\npm-cache\_logs\2020-11-09T12_03_35_739Z-debug.log
```

The problem there is that NPM expects the node_modules to have a subfolder `.staging`, but all the modules are directly in the node_modules folder such as `node_modules\@babel`

--- a/README.md
+++ b/README.md
@@ -73,7 +73,7 @@ npm run build
 When you update `package.json` to include new dependencies, or when you pull changes that affect `package.json`, issue the `npm update` command to download the dependencies.
 
 ## Install Zowe CLI from source
-From your copy of this repository, after a build, issue the following command to install Zowe CLI from source:
+From your copy of this repository, after a build, navigate to the `packages/cli` directory, then issue the following command to install Zowe CLI from source:
 
 ```
 npm install -g

